app-backend/migrations/src_migrations/main/scala/169.scala

```diff
@@ -0,0 +1,29 @@
+import slick.jdbc.PostgresProfile.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M169 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(169)(
+    List(
+      sqlu"""
+ALTER TABLE project_layers
+ADD COLUMN
+  is_single_band boolean NOT NULL DEFAULT false,
+ADD COLUMN
+  single_band_options jsonb;
+
+UPDATE project_layers
+SET
+  is_single_band = projects.is_single_band
```

Comment:
I think we could do something like below to have them done together:
```sql
...
SET
  is_single_band = projects.is_single_band,
  single_band_options = projects.single_band_options
...
```

Reply:
Oh yeah we probably could.